lisp/modules/rename_cues/rename_ui.py

```diff
@@ -0,0 +1,299 @@
+from lisp.ui.ui_utils import translate
+
+
+class RenameUi(QDialog):
+    def __init__(self, parent=None):
+        super().__init__(parent)
+
+        # This list store infos on cues with dict (see self.get_selected_cues)
+        self._cues_list = []
+
+        self.setWindowTitle(translate('RenameCues', 'Rename cues'))
+        self.setWindowModality(Qt.ApplicationModal)
+        self.resize(650, 350)
+
+        self.setLayout(QGridLayout())
+
+        # Preview List
+        self.previewList = QTreeWidget()
+        self.previewList.setColumnCount(2)
+        self.previewList.setHeaderLabels([
+            translate('RenameCues', 'Current'),
+            translate('RenameCues', 'Preview')
+        ])
+        self.previewList.resizeColumnToContents(0)
+        self.previewList.resizeColumnToContents(1)
+        self.previewList.setColumnWidth(0, 300)
+        self.previewList.setSelectionMode(QAbstractItemView.NoSelection)
+        self.previewList.itemPressed.connect(self.onPreviewListItemPressed)
+        self.layout().addWidget(self.previewList, 0, 0, 3, 4)
+
+        # Options buttons
+        self.capitalizeButton = QPushButton()
+        self.capitalizeButton.setText(translate('RenameCues', 'Capitalize'))
+        self.capitalizeButton.clicked.connect(self.onCapitalizeButtonClicked)
+        self.layout().addWidget(self.capitalizeButton, 3, 0)
+
+        self.lowerButton = QPushButton()
+        self.lowerButton.setText(translate('RenameCues', 'Lowercase'))
+        self.lowerButton.clicked.connect(self.onLowerButtonClicked)
+        self.layout().addWidget(self.lowerButton, 4, 0)
+
+        self.upperButton = QPushButton()
+        self.upperButton.setText(translate('RenameCues', 'Uppercase'))
+        self.upperButton.clicked.connect(self.onUpperButtonClicked)
+        self.layout().addWidget(self.upperButton, 5, 0)
+
+        self.removeNumButton = QPushButton()
+        self.removeNumButton.setText(translate('RenameCues', 'Remove Numbers'))
+        self.removeNumButton.clicked.connect(self.onRemoveNumButtonClicked)
+        self.layout().addWidget(self.removeNumButton, 3, 1)
+
+        self.addNumberingButton = QPushButton()
+        self.addNumberingButton.setText(translate('RenameCues', 'Add numbering'))
+        self.addNumberingButton.clicked.connect(self.onAddNumberingButtonClicked)
+        self.layout().addWidget(self.addNumberingButton, 4, 1)
+
+        self.resetButton = QPushButton()
+        self.resetButton.setText(translate('RenameCues', 'Reset'))
+        self.resetButton.clicked.connect(self.onResetButtonClicked)
+        self.layout().addWidget(self.resetButton, 5, 1)
+
+        # Spacer
+        self.layout().addItem(QSpacerItem(30, 0), 3, 2)
+
+        # Regex lines
+        self.outRegexLine = QLineEdit()
+        self.outRegexLine.setPlaceholderText(
+            translate('RenameCues', 'Rename all cue. () in regex below usable with $0, $1 ...'))
+        self.outRegexLine.setToolTip(
```

Comment:
maybe use a small help button (with a "?" icon) to display an dialog with those info. The tooltip will hardy display all this text